examples/tailwindcss/postcss.config.js

```diff
@@ -1,6 +1,7 @@
 module.exports = {
   plugins: [
     require('tailwindcss')('./tailwind.js'),
+    require('postcss-import'),
```

Comment:
It's better to run this plugin first so Tailwind can operate on a single set of CSS after the imports have been inlined, like mentioned in the `postcss-import` documentation:

> This plugin should probably be used as the first plugin of your list. This way, other plugins will work on the AST as if there were only a single file to process, and will probably work as you can expect.

If you tried to `@import` any CSS file that was using Tailwind features like `@responsive`, this setup wouldn't work because Tailwind wouldn't see that code, it would only see an `@import` line. You want all of the imports inlined before running any other plugins.

Reply:
You're right. I didn't see any difference, e.i. I didn't get any errors and my pages looked good. But I didn't check the output CSS.